deps/exokit-bindings/webglcontext/src/webgl.cc

```diff
@@ -4630,6 +4615,22 @@ NAN_METHOD(WebGLRenderingContext::GetParameter) {
       info.GetReturnValue().Set(JS_BOOL(gl->premultiplyAlpha));
       break;
     }
+    case MAX_CLIENT_WAIT_TIMEOUT_WEBGL:
+    case GL_MAX_SERVER_WAIT_TIMEOUT:
+    {
+      info.GetReturnValue().Set(JS_INT(0));
+      break;
+    }
+    case GL_MAX_ELEMENT_INDEX:
+    {
+      GLint64 elementIndex = -1;
+      glGetInteger64v(GL_MAX_ELEMENT_INDEX, &elementIndex);
+      if (glGetError() == GL_NO_ERROR) {
```

Comment:
I am wary of doing any `glGetError` ourselves, because this could stomp on _actual_ unrelated gl errors that the user would want to see. If we capture the error here, and we hit that corner case, this could result in very mysterious and hard-to-debug bugs seen by the user.

Additionally, I think `glGetError` might force a flush of the pipeline, making this a stalling GL call when it might not be otherwise.

Is there some other way to do this without eating the error?

Reply:
agreed.

one possibility: we could skip calling `glGet*` and just return  `0xffffffff` (i.e. just lie).

Here's all that the spec says:

```
GL_MAX_ELEMENT_INDEX
data returns a single value, the maximum index that may be specified during the transfer of generic vertex attributes to the GL.

GL_MAX_ELEMENT_INDEX is accepted by pname only if the GL version is 4.3 or greater.
```
I don't know why it works in Chrome but not Exokit.

Reply:
> GL_MAX_ELEMENT_INDEX is accepted by pname only if the GL version is 4.3 or greater.

This might be it. We are [GL 3.2](https://github.com/webmixedreality/exokit/blob/88d5da063d08959a7b8c314dad61591523116c87/deps/exokit-bindings/glfw/src/glfw.cc#L1131), I think mostly because macOS.